test/libsolidity/syntaxTests/using/using_global_library_for_interface.sol

```diff
@@ -0,0 +1,6 @@
+interface I {}
```

Comment:
I think we could use a bit more coverage with this. What types is this supposed to work for? I see that for example it worked for `uint` on 0.8.13. I think we should have at least this as a test case:
```solidity
using L for uint global;
library L {}
```

But we should really cover more types. For example do we have any tests at all covering `using for global` with enums? Or function pointers?

Reply:
Added some more tests. Enum should work, function pointers not.